tools/mo/openvino/tools/mo/front/common/partial_infer/eltwise.py

```diff
@@ -80,6 +81,34 @@ def broadcast_dims(dim1, dim2):
             node.out_port(0).data.set_value(op(node.out_node().value, values[i + 1]))
 
 
+def eltwise_reverse_infer(node: Node):
+    input_1_shape = node.in_port(0).data.get_shape()
+    input_2_shape = node.in_port(1).data.get_shape()
+    output_shape = node.out_port(0).data.get_shape()
+
+    if output_shape is not None:
+        out_rank = len(output_shape)
+        deduced_in_shape = undefined_shape_of_rank(out_rank)
+
```

Comment:
what if `out_rank` equal to zero? can we handle this case?

Reply:
yes, if `out_rank == 0` `undefined_shape_of_rank` will return valid masked scalar value